src/OrchardCore.Modules/OrchardCore.Widgets/ContentCardShapes.cs

```diff
@@ -0,0 +1,134 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using OrchardCore.ContentManagement;
+using OrchardCore.DisplayManagement.Descriptors;
+
+namespace OrchardCore.Widgets
+{
+    public class ContentCardShapes : IShapeTableProvider
+    {
+        //Card Shapes
+        //private readonly string CONTENT_CARD = "ContentCard";
+        private readonly string CONTENT_CARD_EDIT = "ContentCard_Edit";
```

Comment:
can you remove commented out code if it is unnecesary?
Also by convention we follow the aspnetcore static/constants pattern
so it would be `private const string ContentCardEdit = "ContentCard_Edit", and same for below